Gruntfile.js

```diff
@@ -37,34 +37,20 @@ module.exports = function(grunt) {
       }
     },
 
-    jscs: {
-      src: ['src/**/*.js', '!src/**/enums.js'],
+    eslint: {
       options: {
-        preset: 'google', // as per Google style guide – could use '.jscsrc' instead
-        requireCurlyBraces: ['if']
-      }
-    },
-
-    jshint: {
-      options: {
-        jshintrc: 'build/.jshintrc'
+        configFile: 'build/.eslintrc'
       },
-      // files to validate
-      // can choose more than one name + array of paths
-      // usage with this name: grunt jshint:files
-      files: ['src/**/*.js', '!src/**/enums.js', '!src/**/adapter.js']
+      target: ['src/**/*.js', '!src/**/enums.js', '!src/**/adapter.js' ]
     },
 
     shell: {
       getPythonTestDeps: {
         command: 'python build/get_python_test_deps.py'
       },
-      installPythonTestDepsOnLinux: {
-        command: 'python build/install_webtest_on_linux.py webtest-master/'
-      },
       runPythonTests: {
-        command: ['python', 'build/run_python_tests.py', 'google_appengine/',
-                  out_app_engine_dir, 'webtest-master/'].join(' ')
+        command: ['python', 'build/run_python_tests.py',
+                  'temp/google_appengine/', out_app_engine_dir].join(' ')
```

Comment:
This is the same temp dir as in build/get_python_test_deps.py (which will be a const in that file). Not sure if a constant makes sense here since they are in different files and different languages.